Package.swift

```diff
@@ -7,7 +7,7 @@ let package = Package(
         .library(name: "Bond", targets: ["Bond"])
     ],
     dependencies: [
-        .package(url: "https://github.com/ReactiveKit/ReactiveKit.git", .upToNextMajor(from: "3.7.4")),
+        .package(url: "https://github.com/ReactiveKit/ReactiveKit.git", .upToNextMajor(from: "3.9.0")),
         .package(url: "https://github.com/tonyarnold/Differ.git", .upToNextMajor(from: "1.0.0"))
```

Comment:
You'll want to move Differ up to "1.1.0" for the Xcode 9.3/Swift 4.1 compatibility.